content/blog/trustless-lightning-payments-for-content.md

```diff
@@ -0,0 +1,357 @@
+    github: yzernik
+    twitter: yzernik
+tags:
+  - "Lightning"
+  - "PTLC"
+  - "Bitcoin"
+
+images:
+  - "/post-data/trustless-lightning-payments/ptlc-example-2.png"
+
+appearedfirston:
+    label: "squeaknode.org"
+    url: "https://squeaknode.org/lightning/ptlc/bitcoin/2022/01/07/trustless-lightning"
+
+---
+
+The existence of the Lightning Network enables new types of online communities
+where participation and access to content is dependent on individual users making
+Bitcoin micropayments. These new communities can operate without a central
+authority, and without a middleman taking a cut of the profit.
+
+In this post, I will briefly describe the weaknesses of the existing models
+for Lightning monetization, and how these problems will be solved when PTLC
+payment channels are the standard in Lightning Network.
+
+#### How Lightning incentivizes content
+
+If we imagine a world where social media is decentralized, content will no
+longer be hosted by large companies like Twitter, Facebook, Spotify, and
+Youtube. Instead, we will see individuals and independent hosting providers
+relaying and aggregating content that is signed cryptographically by individual
+users. This raises the question of how will these relays/aggregators be
+incentivized to host content. Lightning payments are the obvious answer.
+
+#### What is wrong with the tipping model?
+
+There are currently attempts to create models of monetization that use
+Lightning network to reward content creators:
+
+- Value-For-Value (Used in Podcasting 2.0 apps like Fountain)
+- Tipping (Used in Zion)
+
+These apps use a voluntary payment model, where any user can access content
+for free, and then decide how much they want to pay in tips to reward the
+content creator.
+
+This leads to a situation where the majority of users are consuming the
+content for free, and a small generous minority of users is generating all
+of the revenue for the content creator. The revenue of the content creator
+is totally dependent on the generosity and altruism of their consumers.
+
+It is the digital equivalent of "busking" on a sidewalk, and hoping that people
+will drop a few coins in your hat.
+
+
+#### Moving to a pay for content model
+
+A better, fairer model would be one where the consumer is required to make a
+payment to get access to the content.
+
+A naive approach to accomplish this would be to simply encrypt the content,
+and then create a Lightning invoice with the decryption key as the invoice
+preimage (using HTLCs), and share the invoice with the consumer. Then, when the consumer
+pays the invoice, they would obtain the decryption key, and use it to unlock
+the content.
+
+<!-- In the current implementation of Lightning in LND (using HTLCs), the preimage -->
+<!-- would be specified in the `r_preimage` field of the `AddInvoice` RPC method. -->
+
+{{< center-figure src="/post-data/trustless-lightning-payments/htlc-example-3.png" caption="" height=600 width=600 >}}
```

Comment:
```suggestion
{{< center-figure src="/post-data/trustless-lightning-payments/htlc-example-3.png" height=600 width=600 >}}
```

or add a caption.

Reply:
Also, seems like there is a lot of transparent padding a the top and bottom. Maybe crop the image to reduce the transparent padding.

![image](https://user-images.githubusercontent.com/19157360/149501722-a741e22c-fe87-4316-b250-c72e23f694a2.png)

Reply:
Fixed.